core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStoreDir.java

```diff
@@ -66,8 +66,11 @@ public void reset() throws IOException {
 
   @Override
   public void scanPages(Consumer<Optional<PageInfo>> pageInfoConsumer) {
+    // Fix thread safety on this iterator or or demise RocksPageStore
```

Comment:
talked to @dbw9580 I think we are going to demise RocksPageStore soon so won't waste time fixing it. This issue is not introduced by this change of mine.